testframework/src/main/java/net/neoforged/testframework/gametest/ParametrizedGameTestSequence.java

```diff
@@ -24,11 +24,25 @@ public ParametrizedGameTestSequence(GameTestInfo info, ExtendedSequence sequence
         this.info = info;
         this.sequence = sequence;
 
+        final AtomicReference<RuntimeException> capturedException = new AtomicReference<>();
```

Comment:
This should catch any `Throwable`.

Reply:
How do you want me to rethrow the `Throwable`? Guava's `Throwables.throwIfUnchecked` and a fallback `new RuntimeException(ex)` for checked exceptions?